spec/controllers/user_tips_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe UserTipsController do
+  let(:user) { create(:user) }
+  let(:tip) { Tip.create }
+  let(:user_tip_params) { {user_id: user, tip_id: tip, result: 'Liked', feedback: 'This tip was good'} }
```

Comment:
I would put all of the user_tip_params as seperate variables (eg
let(:feedback) { "this tip is good" }  }
let(:result) { "liked" }
)

This way you can change the values of all of the parameters throughout the test without having to redefine user_tip_params.  You can simply define user_tip_params as 
let(:user_tip_params) { {user_id: user, tip_id: tip, result: result, feedback: feedback} }